exclude-from.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -e
+
+# Determine the paths to exclude from the deployment
+#
+# Paths are inferred from the REMOTE_PATH environment variable.
+#
+# The function returns two paths: the relative base path for prefixing excluded files,
+# and the relative mu-plugins path.
+determine_exclude_paths() {
+  local base_path="wp-content/"
+  local mu_dir_path="wp-content/mu-plugins/"
+
+  case "$REMOTE_PATH" in
+    "wp-content")
+      base_path=""
+      mu_dir_path="mu-plugins/"
+      ;;
+    "wp-content/mu-plugins")
+      base_path=""
+      mu_dir_path=""
+      ;;
+  esac
```

Comment:
Check me on this, but I believe these represent the only cases in which we would need to modify the paths.

If the user is deploying to other expected directories (eg, `wp-content/plugins` or `wp-content/themes`), the full relative paths are included.

While the above has no negative effect, we could consider further modifying the excludes list at that point to completely remove the WP and WPE-specific file paths. I did consider implementing that, but it felt like overkill 🙃

Reply:
I wonder if `wp-content/` and `wp-content/mu-plugins` (`/` appended) should also be matched 🤔

It seems like the deploy would likely work with either as the `REMOTE_PATH`, so those might need to be added for comparison here.

Reply:
To-do: test with and without appended slash for REMOTE_PATH.